user-integration-guide/symfony-integration.md

```diff
@@ -1,22 +1,27 @@
 # Integrating Opentelemetry PHP into Symfony Applications
 
 ## Introduction 
-As a PHP developer I have always wondered how OpenTelemetry PHP could be beneficial to me. This user guide explains how one can instrument and gain insights from their Symfony application using the OpenTelemetry library. This guide highlights examples on using the OpenTelemtry PHP API. 
-## Set Up
-Ensure you have the following
 
-* This assumes that you have PHP Installed
-* You need [composer](https://getcomposer.org/download/) installed
-* You need [Symfony CLI](https://symfony.com/download) installed 
-* You need [docker](https://docs.docker.com/get-docker/) Installed
+As a developer, you might be wondering how  Telemetry could be beneficial to you. You are not alone.  Without practical examples, the usefulness of distributed tracing can be difficult to grasp for persons without a cloud or site reliability engineering background. This user guide shows how Telemetry could be useful to gain insights into exceptions happening within an application. This example uses the OpenTelemtry PHP library integrated into a Symfony application, bundled with Jaeger and Zipkin, for visualizing data.  
```

Comment:
Did we intend to rename the library to Telemetry, instead of OpenTelemetry?

Reply:
No @beniamin , not yet D: , but OpenTelemetry implements Telemetry I think. We can change it though. If we want.

Reply:
I think mentioning telemetry as it's own separate thing is okay, but I agree with @benagricola that we should use `OpenTelemetry` here.